README.md

```diff
@@ -163,6 +163,7 @@ The component accepts the following props:
 |**`print`**|boolean|true|Show/hide print	 icon from toolbar
 |**`download`**|boolean|true|Show/hide download icon from toolbar
 |**`downloadOptions`**|object||Options to change the output of the CSV file. Default options: `{filename: 'tableDownload.csv', separator: ','}`
+|**`onDownload`**|function||Callback function that triggers when the user downloads the CSV file. `function(buildHead: (columns) => string, buildBody: (data) => string, columns, data) => string`
```

Comment:
Let's add a note here about being able to use this function also to customize the csv output.

Reply:
Done, added in the last commit